bindings/matrix-sdk-ffi/CHANGELOG.md

```diff
@@ -32,6 +32,9 @@ All notable changes to this project will be documented in this file.
 
 ### Breaking changes:
 
+- `TimelineDiff` is now exported as a true `uniffi::Enum` instead of the weird `uniffi::Object` hybrid. This matches
+  both `RoomDirectorySearchEntryUpdate` and `RoomListEntriesUpdate` and can be used in the same way.
```

Comment:
What do you mean here?

Reply:
The exported type of `TimelineDiff` is now the same as `RoomListEntriesUpdate`. It seemed worth mentioning it so that anyone getting compile errors, can just look at how they handle the room list entries diffs and do the same for the timeline diffs.